src/extension/src/controller.ts

```diff
@@ -1,4 +1,6 @@
 import * as vscode from "vscode";
+const os = require("os");
```

Comment:
Is it intentional that we don't use the `import` syntax instead of `require` syntax to get the `os` module? If not, I think we should use the `import` syntax to be consistent with other imports.

Reply:
Yes, it is intentional
https://nodejs.org/api/os.html#os_os

Reply:
@piotrmark what are you thoughts on this?

Reply:
I'd say, let's be consistent.
So `import * as os from "os";`